test/util.js

```diff
@@ -209,3 +210,75 @@ function checkStatsObject ({
     }
   })
 }
+
+/**
+ * @param {Object}   params
+ * @param {Object}   params.t - AVA test object
+ * @param {Object}   params.obj - stats object returned by API
+ * @param {String}   params.groupBy
+ * @param {Object[]} params.results
+ * @param {String}   [order = 'desc']
+ * @param {Function} [additionalResultCheckFn] - if defined, will perform additional check on `result`
```

Comment:
Imprecise.
It seems it is `on each item of results array`.

Reply:
Thanks, I'll update the comment.